src/app/portlets/dot-edit-page/shared/models/layout-grid-box.model.ts

```diff
@@ -0,0 +1,7 @@
+import { NgGridItemConfig } from 'angular2-grid';
+import { Container } from './container.model';
+
+export interface LayoutGridBox {
```

Comment:
I think we need doc here too

something like?

"
Represents a ng-grid block
@link <ng-grid urk> 
"

Reply:
I dont konw it this interface should be into shared because is must be use just into dot-edit-layout-grid.component.ts